.github/workflows/helm-publish.yaml

```diff
@@ -51,6 +63,14 @@ jobs:
               echo "$f"
               helm push $f oci://${REGISTRY,,}
           done
+      - name: Upload the Chart to Rekor
```

Comment:
What is Rekor?

Reply:
is an immutable transparency log, we publish the chart signatures there so everyone can verify